.github/workflows/release.yml

```
@@ -228,7 +228,7 @@ jobs:
   publish:
     name: Publish
     needs: [dist]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

same here since we're only publishing

```suggestion
    runs-on: ubuntu-latest
```